locustempus/settings_travis.py

```diff
@@ -14,7 +14,7 @@
     DATABASES = {
         'default': {
             'ENGINE': 'django.contrib.gis.db.backends.postgis',
-            'NAME': 'test_locustempus',
+            'NAME': 'locustempus',
```

Comment:
why did this change?

Reply:
Oh...was it creating test_test_locustempus?

Reply:
yeah, exactly

Reply:
Looks great - merging!